src/DynamoCore/PackageManager/Package.cs

```diff
@@ -196,12 +196,8 @@ public static Package FromJson(string headerPath, ILogger logger)
         /// <summary>
         /// Load the Package into Dynamo.  
         /// </summary>
-        /// <param name="loader"></param>
+        /// <param name="loadPackageParams"></param>
         /// <param name="logger"></param>
-        /// <param name="libraryServices"></param>
-        /// <param name="context"></param>
-        /// <param name="isTestMode"></param>
-        /// <param name="customNodeManager"></param>
         public void LoadIntoDynamo(LoadPackageParams loadPackageParams, ILogger logger)
```

Comment:
Oops, I like to write documentation but now this makes me feel bad. Thanks @pboyer, I'd fill those one day if I get to this again!